[2024-04-23 03:01:06,206] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dados_climaticos2.extrai_dados scheduled__2024-04-08T00:00:00+00:00 [queued]>
[2024-04-23 03:01:06,210] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dados_climaticos2.extrai_dados scheduled__2024-04-08T00:00:00+00:00 [queued]>
[2024-04-23 03:01:06,210] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2024-04-23 03:01:06,210] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2024-04-23 03:01:06,210] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2024-04-23 03:01:06,221] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): extrai_dados> on 2024-04-08 00:00:00+00:00
[2024-04-23 03:01:06,224] {standard_task_runner.py:52} INFO - Started process 150816 to run task
[2024-04-23 03:01:06,226] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'dados_climaticos2', 'extrai_dados', 'scheduled__2024-04-08T00:00:00+00:00', '--job-id', '17', '--raw', '--subdir', 'DAGS_FOLDER/dados_climaticos.py', '--cfg-path', '/tmp/tmplvix3ghd', '--error-file', '/tmp/tmpwts0xmcc']
[2024-04-23 03:01:06,226] {standard_task_runner.py:80} INFO - Job 17: Subtask extrai_dados
[2024-04-23 03:01:06,253] {task_command.py:370} INFO - Running <TaskInstance: dados_climaticos2.extrai_dados scheduled__2024-04-08T00:00:00+00:00 [running]> on host URATIC9519.RIOBRANCOPETROLEO
[2024-04-23 03:01:06,284] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=dados_climaticos2
AIRFLOW_CTX_TASK_ID=extrai_dados
AIRFLOW_CTX_EXECUTION_DATE=2024-04-08T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-04-08T00:00:00+00:00
[2024-04-23 03:01:07,466] {python.py:173} INFO - Done. Returned value was: None
[2024-04-23 03:01:07,470] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=dados_climaticos2, task_id=extrai_dados, execution_date=20240408T000000, start_date=20240423T060106, end_date=20240423T060107
[2024-04-23 03:01:07,520] {local_task_job.py:156} INFO - Task exited with return code 0
[2024-04-23 03:01:07,532] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
